app/models/dossier.rb

```diff
@@ -468,6 +468,14 @@ def attachments_downloadable?
     !PiecesJustificativesService.liste_pieces_justificatives(self).empty? && PiecesJustificativesService.pieces_justificatives_total_size(self) < Dossier::TAILLE_MAX_ZIP
   end
 
+  def update_with_france_connect(fc_information)
+    self.individual = Individual.create(
```

Comment:
J'ai l'impression qu'on voudrait une erreur si le `create` échoue. 
```suggestion
    self.individual = Individual.create!(
```